backend/src/http/routes/api/backoffice/financeur/financeur-routes.js

```diff
@@ -150,7 +149,7 @@ module.exports = ({ db, middlewares, configuration, logger, postalCodes }) => {
             }
         }
 
-        const finalFilter = { ...filter, ...periodeFilter };
+        const finalFilter = { ...filter, ...lieuFilter, ...periodeFilter };
```

Comment:
attention ça ne marchera pas, le filtre lieuFilter va écraser le $or (dans le cas ou tu filtres sur all), ligne 125

Reply:
il faut utiliser $and donc plutôt  const finalFilter = { $and: [filter, lieuFilter, periodeFilter ] }